bears/general/CPDBear.py

```diff
@@ -0,0 +1,41 @@
+import re
+import os
+
+from coalib.bearlib.abstractions.Lint import Lint
+from coalib.bears.LocalBear import LocalBear
+
+
+class CPDBear(LocalBear, Lint):
+    executable = 'run.sh'
+    arguments = "cpd --minimum-tokens 10" + \
+                " --files {filename}"
+    output_regex = re.compile(
+        r'(?P<message>.*) in .*\n'
+        r'[A-Za-z ]*(?P<line>[0-9]+).*\n')
+
+    def run(self, filename, file):
+        '''
+        Checks duplicate code with ``cpd``.
+        Languages: C++, Java, C#, JavaScript,
+        Fortran, Go, JSP, Objective C,
+        PHP, Python, Ruby, Scala
```

Comment:
Please phrase this out, this text is shown to the user after all. Make it look nicely